### ISSUE TITLE
`reportShadowedImports` regressed between 1.1.282 and 1.1.283

### ISSUE THREAD

**Issue by ottumm** [microsoft/pyright#4324]
This is for the command-line tool, using `reportShadowedImports = true` and the `--warnings` flag.

```toml
# pyproject.toml
[tool.pyright]
pythonVersion = "3.11"
include = ["scd"]
strict = ["scd"]

reportShadowedImports = true
```

```python
# src/copy.py
def copy(): ...
```

```python
# src/main.py
from .copy import copy
```

```
$ npx --yes pyright@1.1.283 --warnings
...
pyright 1.1.283
0 errors, 0 warnings, 0 informations 
Completed in 0.445sec
```

```
$ npx --yes pyright@1.1.282 --warnings
...
pyright 1.1.282
/Users/ottumm/danger/florist/scd/main.py
  /Users/ottumm/danger/florist/scd/main.py:2:1 - error: "/Users/ottumm/danger/florist/scd/copy.py" is overriding the stdlib module "copy" (reportShadowedImports)
1 error, 0 warnings, 0 informations 
Completed in 0.435sec
```

**Comment by rchiodo** [comment]
This is actually on purpose. It was a bug fix for 283. https://github.com/microsoft/pyright/issues/4269

Relative imports shouldn't be marked as an error because they would only work for a local file. You can't load copy from stdlib with 'from .copy import copy' so you must be intending to import the relative one.

**Comment by ottumm** [comment]
Huh. Ok, well Pylance latest version marks this as a warning in VS Code. I'm trying to resolve the discrepancy.

**Comment by rchiodo** [comment]
Pylance latest version prerelease or stable? Prerelease would have the change for 283 I believe. Stable should still have the 282 version of pyright.

**Comment by ottumm** [comment]
Ah, I'm on stable. That probably explains it.

**Comment by ottumm** [comment]
Thanks!
